src/components/settings/SettingsSSOTab/NewSSOConfigCard.jsx

```diff
@@ -0,0 +1,213 @@
+import React, { useContext } from 'react';
+import {
+  Card, Badge, Button, Dropdown, IconButton, Icon, Tooltip, OverlayTrigger,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import {
+  Key, KeyOff, MoreVert,
+} from '@edx/paragon/icons';
+import { SSOConfigContext } from './SSOConfigContext';
+import LmsApiService from '../../../data/services/LmsApiService';
+
+const NewSSOConfigCard = ({
+  config,
+  setLoading,
+  setRefreshBool,
+  refreshBool,
+}) => {
+  const VALIDATED = config.validated_at;
+  const ENABLED = config.active;
+  const CONFIGURED = config.submitted_at < config.configured_at && config.configured_at;
```

Comment:
nit: LTR logic and parentheses for readability.
```suggestion
  const CONFIGURED = config.configured_at && (config.submitted_at < config.configured_at);
```